pom.xml

```diff
@@ -46,6 +46,19 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
Dependencies must be listed in alphabetical order. (Plus empty line in between.)